job-service-db/src/main/resources/procedures/deleteJob.sql

```diff
@@ -25,11 +25,14 @@ DROP FUNCTION IF EXISTS delete_job(
     in_job_id VARCHAR(48),
     in_short_job_id VARCHAR(48)
 );
-CREATE OR REPLACE FUNCTION delete_job(
+DROP FUNCTION IF EXISTS delete_job(
+    in_partition_id VARCHAR(40),
+    in_job_id VARCHAR(48)
+);
+CREATE OR REPLACE PROCEDURE delete_job(
```

Comment:
Have you considered whether the change in semantics going from function to procedure has any other impact? Is there any difference w.r.t what happens when there is an error at different stages?